src/ChanperfTracker.ts

```diff
@@ -0,0 +1,145 @@
+import { EventEmitter } from 'events';
+
+export class ChanperfTracker {
+    constructor() {
+        this.emitter = new EventEmitter();
+        this.filterOutLogs = true;
+        this.chanperfHistory = [];
+    }
+
+    private emitter: EventEmitter;
+    private filterOutLogs: boolean;
+    private chanperfHistory: Array<ChanperfEventData>;
+
+    public on(eventname: 'chanperf-event', handler: (output: ChanperfEventData) => void);
+    public on(eventName: string, handler: (payload: any) => void) {
+        this.emitter.on(eventName, handler);
+        return () => {
+            if (this.emitter !== undefined) {
+                this.emitter.removeListener(eventName, handler);
+            }
+        };
+    }
+
+    private emit(eventName: 'chanperf-event', data: ChanperfEventData) {
```

Comment:
I still think we should omit the `-event` from this name. I know we used `-event` in the rendezvous stuff, but I should have probably caught that too. These ARE events, so you don't need to use the word `event` in their names. Let's just name this event `chanperf`. So `this.emit("chanperf", {...})`;

Reply:
Ok. I'll update both for constancy.

Reply:
Yeah, i guess that makes sense. Technically renaming the redezvous event is a breaking change, but I don't think anyone other than RokuCommunity is using `roku-debug` right now anyway...

Reply:
Yea I don't think this is really an issue. We can bump the version to match this change though. Just to be transparent.